### ISSUE TITLE
0.91 inch isn't working

### ISSUE THREAD

**Issue by Harshu1000-7** [spacehuhn/esp8266_deauther#1734]
> Have you searched for existing (open and closed) issues describing the same problem?

No

**Describe the error**
0.91 inch display won't just start working

```
Error/Compile/Output Log
Nothing 
```

**To Reproduce**

**Expected behavior**
Expected to see display output and showing attacks "death probe and beacon"

**Screenshots**

**Environment (please complete the following information):**

 - Hardware [e.g. DSTIKE, DIY]
 - esp8266

**Attempts**
Changing pin numbers

**Additional context**
I've seen videos of people directly plugging 0.91 inch display after unplugging 0.96 inch ssd1306 display the 0.91 inch as mentioned on data sheet has the same I ssd1306 but still the display won't work

**Comment by Sudo-CHOPP** [comment]
Any luck? I cant get mine working either. Im brand spanking new to this kind of stuff though so I have no idea what I'm doing. Just kinda tinkering.

**Comment by Harshu1000-7** [comment]
> Any luck? I cant get mine working either. Im brand spanking new to this kind of stuff though so I have no idea what I'm doing. Just kinda tinkering.

yes i got the fix, you have to connect the display the way it is, and the display can be enabled by going on webserver settings and enabling it, its disabled by default
